php/deprecated.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Deprecated functions.
+ *
+ * Deprecated methods can also appear as functions here, with the format namespace__class__method().
+ *
+ * @see Block_Lab\Component_Abstract->_call()
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2018, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+/**
+ * Show a PHP error to warn developers using deprecated functions.
+ *
+ * @param string $function    The function that was called.
+ * @param string $version     The version of Block Lab that deprecated the function.
+ * @param string $replacement The function that should have been called.
+ */
+function block_lab_deprecated_function( $function, $version, $replacement ) {
+	_deprecated_function(
+		// filter_var is used for sanitization here as it allows arrow functions ("->").
+		filter_var(
```

Comment:
What do you think about passing something like `FILTER_SANITIZE_ENCODED` as the 2nd argument of `filter_var()`? 

Good idea to use that function. But so far, it looks like it doesn't sanitize much. 

For example, `filter_var( '<script>doThis();</script>' )` doesn't remove anything.

I'm not sure if `FILTER_SANITIZE_ENCODED` is the right argument, though.

Reply:
Great thinking! I went with `FILTER_SANITIZE_STRING` in the end, as:

- It allows arrow functions without encoding them (`->`)
- It removes `<script>` and other malicious things
- We use it elsewhere in the plugin

Reply:
Implemented in 7b98dd7.

Reply:
Nice, good idea.

Reply:
Like you mentioned, I also see how passing `FILTER_SANITIZE_STRING` removes `<script>`, but doesn't remove things we need like `->` or `()`